### ISSUE TITLE
Allow a blank value for docroot if the first value is wrong in ddev config

### ISSUE THREAD

**Issue by TopherGopher** [drud/ddev#97]
## What happened (or feature request):
When running `ddev config`, I entered a folder name the first time that didn't exist. I decided to set the root instead back to the top-level of the project, but couldn't submit a blank input.

## What you expected to happen:
I should have been able to submit a blank input denoting that the docroot was in the project root folder.

## How to reproduce it (as minimally and precisely as possible):
Run `ddev config`, enter a folder name that doesn't exist, then try to set the docroot to <blank>.

## Anything else do we need to know:
While most people may not type an invalid folder name then no folder name intentionally, you could still typo a single letter into the input field and be unable to submit the root as docroot.

## Related source links or issues

**Command Output:**
```
➜  drud-d8 ddev config
Creating a new ddev project config in the current directory (/Users/csterling/src/dummy/drud-d8)
Once completed, your configuration will be written to /Users/csterling/src/dummy/drud-d8/.ddev/config.yaml
Project name (drud-d8):
The docroot is the directory from which your site is served. This is a relative path from your application root (/Users/csterling/src/dummy/drud-d8)
You may leave this value blank if your site files are in the application root
Docroot Location: docroot
No directory could be found at /Users/csterling/src/dummy/drud-d8/docroot. Please enter a valid docroot
The docroot is the directory from which your site is served. This is a relative path from your application root (/Users/csterling/src/dummy/drud-d8)
You may leave this value blank if your site files are in the application root
Docroot Location (docroot):
No directory could be found at /Users/csterling/src/dummy/drud-d8/docroot. Please enter a valid docroot
The docroot is the directory from which your site is served. This is a relative path from your application root (/Users/csterling/src/dummy/drud-d8)
You may leave this value blank if your site files are in the application root
Docroot Location (docroot):
No directory could be found at /Users/csterling/src/dummy/drud-d8/docroot. Please enter a valid docroot
The docroot is the directory from which your site is served. This is a relative path from your application root (/Users/csterling/src/dummy/drud-d8)
You may leave this value blank if your site files are in the application root
Docroot Location (docroot): /
Application Type [drupal7, drupal8, wordpress]:
```

**Comment by tannerjfco** [comment]
I don't believe this issue occurs at this point. I have been able to provide no input for the docroot prompt, and config succeeds, detecting my apptype. I'm going to close - @TopherGopher please feel free to re-open if you think this issue is not resolved.

**Comment by rfay** [comment]
No, this issue still exists:

https://asciinema.org/a/1t8z7cvht7yozoeoe4ng9mz97

**Comment by tannerjfco** [comment]
Fixed in #185
